app/components/candidate_interface/find_down_course_choices_banner.html.erb

```diff
@@ -0,0 +1,15 @@
+<% unless CandidateInterface::CanAddCourseChoice.can_add_course_choice?(application_form: @application_form) %>
```

Comment:
Should this check be `EndOfCycleTimetable.find_down?`, given the subtle logical difference between that and `can_add_course_choice?`

Reply:
I think the banner should actually be renamed to candidate_cannot_add_courses_banner.rb or something. 

It's a bit of a moot point anyway though as I just looked at the designs and you there's no longer a link to get to the courses section. It now looks like this:

![image](https://user-images.githubusercontent.com/42515961/89898860-547bb100-dbd9-11ea-874f-b73014a1fd5c.png)